easybuild/easyconfigs/u/UCX/UCX-1.5.0rc1-GCCcore-8.2.0-hpcx.eb

```diff
@@ -0,0 +1,47 @@
+# Note:
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+easyblock = 'ConfigureMake'
+
+name = 'UCX'
+version = '1.5.0rc1'
+versionsuffix = '-hpcx'
+
+homepage = 'http://www.openucx.org/'
+description = """Unified Communication X
+An open-source production grade communication framework for data centric
+and high-performance applications
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+toolchainopts = {'pic': True}
+
+sources = [{
+    'filename': 'ucx-1.5.0rc1.tar.gz',
+    'download_filename': 'ucx-1.5.0.tar.gz'
+}]
+source_urls = ['https://github.com/openucx/ucx/releases/download/v1.5.0-rc1']
```

Comment:
Source_urls should come before sources line

Reply:
Fixed.